### DISCUSSION TITLE
Bundling script

### DISCUSSION THREAD

**Discussion by maapple** [ghiscoding/slickgrid-universal#1419] [General]
Thank you for providing such a great framework. I would like to use it, but currently I am not using a frontend-backend separation development pattern. Is it possible to provide a bundling script so that I can directly include it in a <script> tag and write JavaScript to use it?

**Comment by ghiscoding** [comment]
You can try the zip that is bundled for Salesforce [here](https://github.com/ghiscoding/slickgrid-universal/tree/master/packages/vanilla-force-bundle/dist-grid-bundle-zip), this is also a duplicate of this Discussion #765
